### ISSUE TITLE
Packing sequence suggestion

### ISSUE THREAD

**Issue by niyongsheng** [skjolber/3d-bin-container-packing#792]
* 1、packing order be configurable, from bottom up or from rear to front；
* 2、same box can be arranged next to each other；
* 3、box add the maximum number of layers that are allowed to be stacked on top；
![1699265388035](https://github.com/skjolber/3d-bin-container-packing/assets/28373699/5b88d58e-24b7-4d31-95f3-f997fb46e3ca)

**Comment by skjolber** [comment]
Hi, 

bullet 3 would require weight calculation.

**Comment by woodward54** [comment]
+1

packing order would be useful for our use case as well

**Comment by skjolber** [comment]
So packaging sequence as in

1. place items x,y,x first, then items a,b,c
2. prefer stacking at free points according to their z or x coodinate

**Comment by woodward54** [comment]
Yup, use case is having a box loading order preference for a truck. i.e. box group 1 gets loaded first, then group 2, ect.
